docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -6,7 +6,6 @@ labels:
   products:
     - cloud
     - enterprise
-    - oss
```

Comment:
we don't mention oss anymore?

Reply:
Public dashboards are in OSS and they are in public preview.
Email sharing is for Cloud and Enterprise; this particular feature is in private preview (not public preview) like we have below. Can we change that too?